src/libos/src/net/socket/unix/stream/address_space.rs

```diff
@@ -7,9 +7,28 @@ lazy_static! {
     pub(super) static ref ADDRESS_SPACE: AddressSpace = AddressSpace::new();
 }
 
+#[derive(PartialEq, Eq, PartialOrd, Ord)]
+pub enum AddressSpaceKey {
+    FileKey(usize),
+    AbstrKey(String),
+    NoExist,
```

Comment:
`NoExsit` seems to be useless, should we remove it and use `None` for the return value of `get_key`? If `get_key` returns `None`, should we return the Error?

Reply:
removed.